worlds/Files.py

```diff
@@ -56,18 +56,18 @@ def __new__(mcs, name: str, bases: Tuple[type, ...], dct: Dict[str, Any]) -> Aut
 
     @staticmethod
     def get_handler(game: str) -> Union[AutoPatchExtensionRegister, List[AutoPatchExtensionRegister]]:
-        for extension_type, handler in AutoPatchExtensionRegister.extension_types.items():
-            if extension_type == game:
-                if handler.required_extensions:
-                    handlers = [handler]
-                    for required in handler.required_extensions:
-                        if required in AutoPatchExtensionRegister.extension_types:
-                            handlers.append(AutoPatchExtensionRegister.extension_types[required])
-                        else:
-                            raise NotImplementedError(f"No handler for {required}.")
-                    return handlers
-                else:
-                    return handler
+        if game in AutoPatchExtensionRegister.extension_types:
+            handler = AutoPatchExtensionRegister.extension_types.get(game)
```

Comment:
I was thinking that the fact get returns a default helped avoid an if check but I don't see it anymore - given that, the normal index notation should also be fine. Not sure how much core maintainers care about one over the other but I don't

Reply:
It looks like it could be
```python
handler = AutoPatchExtensionRegister.extension_types.get(game, APPatchExtension)
```
without the `if` above it.

Reply:
I think it's because Pycharm shows the regular version for `Mapping.get` rather than the overload with the additional argument if you aren't actively typing in another argument.